jicoco/src/main/kotlin/org/jitsi/health/HealthChecker.kt

```diff
@@ -168,9 +169,9 @@ class HealthChecker(
         }
 
         if (exception == null) {
-            logger.debug(
+            val message =
                 "Performed a successful health check in $checkDuration. Sticky failure: ${stickyFailures && hasFailed}"
-            )
+            if (previousResult == null) logger.debug(message) else logger.info(message)
```

Comment:
Shouldn't this be `if (previousResult)`?

Reply:
No, this.lastResult and previousResult are `null` if successful, or an exception otherwise